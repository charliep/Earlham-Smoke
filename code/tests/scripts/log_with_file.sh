#!/bin/sh

# log_with_file.sh
# 
#
# Created by Gilbert Bailey on 3/23/10.
# Copyright 2010 Earlham College. All rights reserved.
EXPECTED_ARGS=1

if [ $# -ne $EXPECTED_ARGS ]
then
  echo "Usage: `basename $0` <file.out>"
  exit $E_BADARGS
fi

OUTPUT=$1

../../Smoke -l log_test > outputs/$OUTPUT

exit 0


