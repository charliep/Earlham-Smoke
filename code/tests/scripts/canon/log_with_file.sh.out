logging is set, logfile: log_test
Smoke.gdf
Smoke!!!
